reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md

```diff
@@ -301,123 +507,167 @@ Aliases:
 Accept wildcard characters: True
 ```
 
 ### -MaxSamples
-Specifies the number of samples to get from each counter.
-The default is 1 sample.
-To get samples continuously (no maximum sample size), use the Continuous parameter.
 
-To collect a very large data set, consider running a Get-Counter command as a Windows PowerShell background job.
-For more information, see about_Jobs and Start-Job.
+Specifies the number of samples to get from each specified performance counter. To get a constant
+stream of samples, use the **Continuous** parameter.
+
+If the **MaxSamples** parameter isn't specified, `Get-Counter` only gets one sample for each
+specified counter.
+
+To collect a large data set, run `Get-Counter` as a PowerShell background job. For more information,
+see [about_Jobs](../Microsoft.PowerShell.Core/About/about_Jobs.md).
 
 ```yaml
-Type: Int64
+Type: Long
 Parameter Sets: GetCounterSet
 Aliases:
 
 Required: False
 Position: Named
-Default value: 1
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -SampleInterval
-Specifies the time between samples in seconds.
-The minimum value and the default value are 1 second.
+
+Specifies the number of seconds between samples for each specified performance counter. If the
+**SampleInterval** parameter isn't specified, `Get-Counter` uses a one-second interval.
 
 ```yaml
-Type: Int32
+Type: Int
 Parameter Sets: GetCounterSet
 Aliases:
 
 Required: False
 Position: Named
-Default value: 1
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
 ## INPUTS
 
 ### System.String[]
-You can pipe counter paths and counter set (ListSet) names to Get-Counter.
+
+`Get-Counter` accepts pipeline input for counter paths and counter set names.
+
 ## OUTPUTS
 
-### Microsoft.PowerShell.Commands.GetCounter.CounterSet, Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSampleSet, Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSample
-The ListSet parameter gets Microsoft.PowerShell.Commands.GetCounter.CounterSet objects.
-The Counter parameter gets Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSampleSet objects.
-Each counter value is a Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSample object.
+### Output objects
```

Comment:
The H3 here needs to list the type names of the output object. So please restore the previous text.

Reply:
resolved.